lib/plug/adapters/cowboy2.ex

```diff
@@ -34,13 +28,18 @@ defmodule Plug.Adapters.Cowboy2 do
       Defaults to 5000ms.
 
     * `:protocol_options` - Specifies remaining protocol options,
-      see [Cowboy docs](https://ninenines.eu/docs/en/cowboy/2.0/manual/cowboy_http/).
-
-  All other options are given to the underlying transport. When running
-  on HTTPS, any SSL configuration should be given directly to the adapter.
-  See `https/3` for an example and read `Plug.SSL.configure/1` to understand
-  about our SSL defaults. When using a unix socket, OTP 21+ is required for `Plug.Static`
-  and `Plug.Conn.send_file/3` to behave correctly.
+      see [Cowboy docs](https://ninenines.eu/docs/en/cowboy/2.5/manual/cowboy_http/).
+
+    * `:transport_options` - A keyword list specifying transport options,
+        see [ranch docs](https://ninenines.eu/docs/en/ranch/1.6/manual/ranch/).
+        By default `num_acceptors` will be set to `100` and `max_connections` to
+        `16_384`.
+
+  All other options are given to the underlying transport as socket_options When
```

Comment:
Minor things:

1. Missing dot before “When”
2. Looking at the PR, it seems the name is `socket_opts` and not `socket_options`
3. I would use backticks and the atom representation `:socket_opts`

🙂